pkg/subctl/operator/lighthouse/scc/ensure.go

```diff
@@ -0,0 +1,83 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scc
```

Comment:
Why are we copy&pasting this instead of refactoring the use of: https://github.com/submariner-io/submariner-operator/blob/master/pkg/subctl/operator/submarinerop/scc/ensure.go  into a common function ?

Reply:
We did decouple Lighthouse related things to independent functions. The service-account role and role biding follow this pattern.

Reply:
Yeah, but this is basically the same function, only the "service account" part differs, could we make a common function that we call from both ensures and avoid the replicated code?

Reply:
A generic "updateServiceAccount" can be written and called from different ensures. Let us get this in for now to unblock and then refactor. Change will be required in existing code to use the new function, so better do it in new PR. I can work on it once this and my helm changes go in.

Reply:
Ok, but let's do it asap, otherwise we will end maintaining 2 versions of the same code.